libbpf-tools/schedsnoop.h

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Alibaba Cloud
+#ifndef SCHEDSNOOP_H
+#define SCHEDSNOOP_H
+
+#define NR_ENTRY_MAX		10000
+#define NR_CPU_MAX		512

```

Comment:
you don't seem to use it, just for that unnecessary check

Reply:
NR_ENTRY_MAX is used for setting the max entry of syscall_info_maps.

Reply:
that comment is specifically for NR_CPU_MAX